HelpSource/Classes/PbindefCut.schelp

```diff
@@ -0,0 +1,51 @@
+class:: PbindefCut
```

Comment:
PbindefCut.mergeNewPairs is undocumented

Reply:
now private.